themes/hugo-refresh/layouts/_default/_markup/render-image.html

```diff
@@ -0,0 +1,4 @@
+<figure>
+    <img src="{{ .Destination | safeURL }}" alt="{{ .Text }}">
+    <figcaption>{{ .Title }}</figcaption>
+  </figure>
```

Comment:
could you please fix the indentation an ensure there is a newline at the end of the file?
```suggestion
    <img src="{{ .Destination | safeURL }}" alt="{{ .Text }}">
    <figcaption>{{ .Title }}</figcaption>
</figure>

```